kuma/wiki/views/document.py

```diff
@@ -865,106 +826,56 @@ def react_document(request, document_slug, document_locale):
             }), extra_tags='wiki_redirect')
         return HttpResponsePermanentRedirect(url)
 
-    # Read some request params to see what we're supposed to do.
-    rendering_params = {}
-    for param in ('raw', 'summary', 'include', 'edit_links'):
-        rendering_params[param] = request.GET.get(param, False) is not False
-    rendering_params['section'] = request.GET.get('section', None)
-    rendering_params['render_raw_fallback'] = False
+    # Get the SEO summary
+    seo_summary = doc.get_summary_text()
 
-    # Are we in a content experiment?
-    original_doc = doc
-    doc, exp_params = _apply_content_experiment(request, doc)
-    rendering_params['experiment'] = exp_params
+    # Get the additional title information, if necessary.
+    seo_parent_title = _get_seo_parent_title(
+        doc, slug_dict, document_locale)
 
-    # Get us some HTML to play with.
-    rendering_params['use_rendered'] = (
-        kumascript.should_use_rendered(doc, request.GET))
-    doc_html, ks_errors, render_raw_fallback = _get_html_and_errors(
-        request, doc, rendering_params)
-    rendering_params['render_raw_fallback'] = render_raw_fallback
-
-    # Start parsing and applying filters.
-    if doc.show_toc and not rendering_params['raw']:
-        toc_html = doc.get_toc_html()
+    # Record the English slug in Google Analytics,
+    # to associate translations
+    if doc.locale == 'en-US':
+        en_slug = doc.slug
+    elif doc.parent_id and doc.parent.locale == 'en-US':
+        en_slug = doc.parent.slug
     else:
-        toc_html = None
-    doc_html = _filter_doc_html(request, doc, doc_html, rendering_params)
+        en_slug = ''
 
-    if rendering_params['raw']:
-        response = _document_raw(doc_html)
-    else:
-        # Get the SEO summary
-        seo_summary = doc.get_summary_text()
```

Comment:
All the rendering_params code here and above seems like wiki-related functionality that we don't need for the static site.